### DISCUSSION TITLE
Fail to create TALDatePickerDialog (the demo ALFmxControl running as well)

### DISCUSSION THREAD

**Discussion by pejuangdelphi** [MagicFoundation/Alcinoe#187] [General]
I try to write code in my new demo project:

`ALFreeAndNil(FDatePickerDialog);
FDatePickerDialog := TALDatePickerDialog.Create('OK', // const aBtnOKCaption: string;
  'Cancel', // const aBtnCancelCaption: string;
  '', // const aBtnClearCaption: string
  'this is a title'); // const aTitle: String = ''
FDatePickerDialog.Show(YearOf(NOW), // const aYear: integer;
  MonthOf(NOW), // const aMonth: integer;
  DayOfTheMonth(NOW)); // const aDayOfMonth: integer);`

After running the app, I still saw the standard calendar... 

![Screenshot_2021-05-17-06-44-04-660_com embarcadero Project18](https://user-images.githubusercontent.com/45804972/118416774-76134700-b6db-11eb-8e83-3bb096c12fbe.jpg)

I have add library to my project:

![image](https://user-images.githubusercontent.com/45804972/118416928-1cf7e300-b6dc-11eb-8a44-b50f4105a53b.png)

then also add some path to the project

![image](https://user-images.githubusercontent.com/45804972/118416993-6811f600-b6dc-11eb-8590-bf2484ecabe1.png)

**Comment by Zeus64** [comment]
You need to include the styles.xml (look in ALFmxControls demo), especialy line like this one :

`  <style name="AppThemeDatePickerDialogTheme" parent="@android:style/Theme.Material.Light.Dialog.Alert">
    <item name="android:datePickerStyle">@style/AppThemeDatePickerStyle</item>
    <item name="android:colorAccent">#ff0288d1</item>
    <item name="android:colorControlNormal">#ff6bc6e7</item>
  </style>

  <style name="AppThemeDatePickerStyle" parent="@android:style/Widget.Material.Light.DatePicker">
    <item name="android:datePickerMode">spinner</item>
  </style>`

**Comment by pejuangdelphi** [comment]
@Zeus64 

Thank you so much... Running well now....
